src/components/Dashboard/Currency/Currency.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
+import currencyApi from "./api/currencyApi";
+import LoaderSpinner from "../../_General/Loader";
+import styles from "./Currency.module.css";
+
+export default function CurrencyTable() {
+  const [currency, setCurrency] = useState([]);
+
+  useEffect(() => {
+    const fetchCurrency = async () => {
```

Comment:
może lepiej zmienną nazwać inaczej niż funkcję, której używasz w środku boję się, że może się to skończyć czasem jakimś bugiem, a na pewno będzie lepiej dla czytelności